### DISCUSSION TITLE
Neotree missing commands

### DISCUSSION THREAD

**Discussion by timjackleus** [LazyVim/LazyVim#922] [Q&A] [answered]
I'm in the process of porting some of my old config into LazyVim and want to use `:NeoTreeFloatToggle` or `:NeotreeFocus` instead of the current `Neotree` command  ([reason here](https://github.com/nvim-neo-tree/neo-tree.nvim/issues/902)) but the command does not seem to exist

vanilla setup:
![image](https://github.com/LazyVim/LazyVim/assets/5394335/044b09e2-e90c-428a-9a68-2b4e5757e2b0)
in LazyVim:
![image](https://github.com/LazyVim/LazyVim/assets/5394335/5e9138e7-018f-4002-ab1a-bb6c8b75dc5a)

What am I missing?

**Comment by folke** [comment, marked as answer]
LazyVim has this option enabled.
`vim.g.neo_tree_remove_legacy_commands = 1`

The commands in your first screenshot are all deprecated.

**Reply by timjackleus** [reply]
Completely missed that one, thank you!

...And thank you for doing so much for this community!
